Bio/motifs/__init__.py

```diff
@@ -25,15 +25,15 @@ def create(instances, alphabet=None):
 
 def parse(handle, format):
     """Parse an output file from a motif finding program.
-
     Currently supported formats (case is ignored):
-     - AlignAce:      AlignAce output file format
-     - MEME:          MEME output file motif
-     - MAST:          MAST output file motif
-     - TRANSFAC:      TRANSFAC database file format
-     - pfm:           JASPAR-style position-frequency matrix
-     - jaspar:        JASPAR-style multiple PFM format
-     - sites:         JASPAR-style sites file
+        - AlignAce:      AlignAce output file format
+        - MEME:          MEME output file motif
+        - MINIMAL:       MINIMAL MEME output file motif
+        - MAST:          MAST output file motif
+        - TRANSFAC:      TRANSFAC database file format
+        - pfm:           JASPAR-style position-frequency matrix
+        - jaspar:        JASPAR-style multiple PFM format
+        - sites:         JASPAR-style sites file
```

Comment:
The indentation change is breaking the RST validation,

```
Bio/motifs/__init__.py:29:1: RST301 Unexpected indentation.
```

Can you go back to the original indentation here please (while adding the bullet point about MINIMAL MEME);